fs/fs.go

```diff
@@ -78,9 +78,10 @@ func NewFsInfo(context Context) (FsInfo, error) {
 	}
 	supportedFsType := map[string]bool{
 		// all ext systems are checked through prefix.
-		"btrfs": true,
-		"xfs":   true,
-		"zfs":   true,
+		"btrfs":   true,
+		"overlay": true,
```

Comment:
This will result in all the overlay filesystems being created for containers from being included in the list of filesystems on the host. It will make the API pretty much unusable.